projects/ad9656_fmc/zcu102/Makefile

```diff
@@ -0,0 +1,28 @@
+####################################################################################
+## Copyright 2018(c) Analog Devices, Inc.
+## Auto-generated, do not modify!
+####################################################################################
+
+PROJECT_NAME := ad9656_zcu102
+
+M_DEPS += ../common/ad9656_bd.tcl
+M_DEPS += ../../common/zcu102/zcu102_system_constr.xdc
+M_DEPS += ../../common/zcu102/zcu102_system_bd.tcl
+M_DEPS += ../../../library/xilinx/common/ad_iobuf.v
+M_DEPS += ../../../library/util_cdc/sync_bits.v
+M_DEPS += ../../../library/jesd204/scripts/jesd204.tcl
+#M_DEPS += ../../../library/common/util_pulse_gen.v
+#M_DEPS += ../../../library/common/ad_bus_mux.v
```

Comment:
commented code can be removed. The makefile should probably created automatically with the script

Reply:
Deleted the commented code